lib/rspec/openapi/record_builder.rb

```diff
@@ -78,13 +80,22 @@ def find_rails_route(request, app: Rails.application, fix_path: true)
       request.path_info = File.join(request.script_name, request.path_info)
     end
 
-    app.routes.router.recognize(request) do |route|
-      unless route.path.anchored
-        route = find_rails_route(request, app: route.app.app, fix_path: false)
+    found_route = nil
+    if app.respond_to? :routes
```

Comment:
The tests don't fail even if you don't check `if app.respond_to? :routes`. Please remove this check or add a test that fails without this check.

Reply:
addressed in https://github.com/k0kubun/rspec-openapi/pull/38/commits/81751cb19734dbe686d98aa357843be3e3c3acf1